.eslintrc.js

```diff
@@ -25,5 +25,6 @@ module.exports = {
       'error',
       { blankLine: 'always', prev: '*', next: 'return' },
     ],
+    'no-console': ['warn', { allow: ['warn', 'error'] }],
```

Comment:
For console and debugger I would prefer to use `error` since some devs are ignoring warnings and may leave logs with some sensitive data (or data that users should not know :D)

Reply:
Well, I'm using console.log for debugging sometimes and it builds with error what makes development difficult. I prefer to deny warnings in CI. So I'll leave it as it is if you don't mind.

Reply:
As you wish. To debug app you can also use `debugger` command, sometimes it is even simpler